packages/expo-router/src/testing-library/require-context-ponyfill.ts

```diff
@@ -3,7 +3,12 @@ import fs from 'node:fs';
 // @ts-ignore: types node
 import path from 'node:path';
 
-import { RequireContext } from '../types';
+import type { RequireContext } from '../types';
+
+interface RequireContextPonyFill extends RequireContext {
+  __add(file: string): void;
+  __delete(file: string): void;
+}
```

Comment:
I don't want to rebuild the context every time a file changes, so I added these methods to modify the context.